homeworks/7/InstagramApi/instagramhw/endpoints/PostsServlet.java

```diff
@@ -0,0 +1,169 @@
+package com.example.instagramhw.endpoints;
+
+import com.example.instagramhw.models.Comment;
+import com.example.instagramhw.models.Post;
+import com.example.instagramhw.models.ReadRequest;
+import com.example.instagramhw.models.User;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+
+import java.io.IOException;
+
+@WebServlet(name = "PostsServlet", value = "/posts/*")

```

Comment:
You have 2 servlets named almost identically. You should've have thought of a better way to split the logic.